### PR TITLE
SL-1709 Paths consistency

### PR DESCRIPTION
**Pull request by StefanDywersant**
### Please check if the PR fulfills these requirements
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### What kind of change does this PR introduce?

Bug fix

### What is the current behavior?

https://stoplightio.atlassian.net/browse/SL-1709

### Does this PR introduce a breaking change?

All ValidationResults has been annotated with paths as arrays of path fragments. Except schema ValidationResults (those provided by ajv). These were arrays of path fragments encoded with json pointer.

Currently all ValidationResults are returned as just path arrays without json encoding.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,6 +44,7 @@
     "test.watch": "yarn test --watch"
   },
   "dependencies": {
+    "@stoplight/json": "^1.5.0",
```
Comment (chris-miaskowski) [inline package.json]: 1.5.x ;)

**Comment by marbemac** [comment]
We have a `next` branch in this repo - please target that!

File: package.json (hunk 1 above)
Comment (StefanDywersant) [reply package.json]: 👍

**Comment by StefanDywersant** [comment]
> We have a `next` branch in this repo (for now, might get rid of it since spectral is fairly stable right now) - please target that! Otherwise 👍.

Can we somehow release spectral to npm? Graph's diagnostics consistency depends on that fix.

**Pull request merged**